content/docs/reference-glossary.md

```diff
@@ -150,7 +150,7 @@ Don't pass something like `Math.random()` to keys. It is important that keys hav
 
 ## [Refs](/docs/refs-and-the-dom.html)
 
-React supports a special attribute that you can attach to any component. The `ref` attribute can be a string or a callback function. When the `ref` attribute is a callback function, the function receives the underlying DOM element or class instance (depending on the type of element) as its argument. This allows you to have direct access to the DOM element or component instance.
+React supports a special attribute that you can attach to any component. The `ref` attribute can be a string or a callback function or created by [`React.createRef()` function](/docs/react-api.html#reactcreateref). When the `ref` attribute is a callback function, the function receives the underlying DOM element or class instance (depending on the type of element) as its argument. This allows you to have direct access to the DOM element or component instance.
```

Comment:
The new style refs should go first.

“An object created by calling `React.createRef()`, a function, or a string (in legacy API).”